infrastructure/clear-all-data.sh

```diff
@@ -0,0 +1,107 @@
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+
+set -e
+
+print_usage_and_exit () {
+    echo 'Usage: ./clear-all-data.sh REPLICAS ENV'
+    echo ""
+    echo "If your MongoDB is password protected, an admin user's credentials can be given as environment variables:"
+    echo "MONGODB_ADMIN_USER=your_user MONGODB_ADMIN_PASSWORD=your_pass"
+    echo ""
+    echo "If your Elasticsearch is password protected, an admin user's credentials can be given as environment variables:"
+    echo "ELASTICSEARCH_ADMIN_USER=your_user ELASTICSEARCH_ADMIN_PASSWORD=your_pass"
+    exit 1
+}
+
+if [ -z "$1" ] ; then
+    echo 'Error: Argument REPLICAS is required in position 1.'
+    print_usage_and_exit
+fi
+
+if [ -z "$2" ] ; then
+    echo 'Error: Argument ENV is required in position 2.'
+    print_usage_and_exit
+fi
+
+REPLICAS=$1
+ENV=$2
+
+if [ "$REPLICAS" = "0" ]; then
+  HOST=mongo1
+  NETWORK=opencrvs_default
+  echo "Working with no replicas"
+elif [ "$REPLICAS" = "1" ]; then
+  HOST=rs0/mongo1
+  NETWORK=opencrvs_overlay_net
+  echo "Working with 1 replica"
+elif [ "$REPLICAS" = "3" ]; then
+  HOST=rs0/mongo1,mongo2,mongo3
+  NETWORK=opencrvs_overlay_net
+  echo "Working with 3 replicas"
+elif [ "$REPLICAS" = "5" ]; then
+  HOST=rs0/mongo1,mongo2,mongo3,mongo4,mongo5
+  NETWORK=opencrvs_overlay_net
+  echo "Working with 5 replicas"
+else
+  echo "Script must be passed an understandable number of replicas: 0,1,3 or 5"
+  exit 1
+fi
+
+mongo_credentials() {
+  if [ ! -z ${MONGODB_ADMIN_USER+x} ] || [ ! -z ${MONGODB_ADMIN_PASSWORD+x} ]; then
+    echo "--username $MONGODB_ADMIN_USER --password $MONGODB_ADMIN_PASSWORD --authenticationDatabase admin";
+  else
+    echo "";
+  fi
+}
+
+elasticsearch_host() {
+  if [ ! -z ${ELASTICSEARCH_ADMIN_USER+x} ] || [ ! -z ${ELASTICSEARCH_ADMIN_PASSWORD+x} ]; then
+    echo "$ELASTICSEARCH_ADMIN_USER:$ELASTICSEARCH_ADMIN_PASSWORD@elasticsearch:9200";
+  else
+    echo "elasticsearch:9200";
+  fi
+}
+
+drop_database () {
+  local database=${1}
+  docker run --rm --network=$NETWORK mongo:4.4 mongo $database $(mongo_credentials) --host $HOST --eval "db.dropDatabase()"
+}
+
+# Delete all data from mongo
+#---------------------------
+drop_database hearth-dev;
+
+drop_database openhim-dev
+
+drop_database user-mgnt;
+
+drop_database application-config
+
+drop_database metrics
```

Comment:
we need to add the `performance` db here

Reply:
Done.